mmocr/utils/ocr.py

```diff
@@ -400,6 +420,99 @@ def __init__(self,
             if hasattr(model, 'module'):
                 model = model.module
 
+    @staticmethod
+    def tesseract_det_inference(imgs):
+        """Inference image(s) with the tesseract detector.
+
+        Args:
+            imgs (list[ndarray]): images to inference.
+
+        Returns:
+            result (dict): Predicted results.
+        """
+        assert is_type_list(imgs, np.ndarray)
+
+        # Very very gross way to find tessdata :(
+        if sys.platform == 'linux':
+            api = PyTessBaseAPI()
+        elif sys.platform == 'win32':
+            try:
+                p = subprocess.Popen(
+                    'where tesseract', stdout=subprocess.PIPE, shell=True)
+                s = p.communicate()[0].decode('utf-8').split('\\')
+                path = s[:-1] + ['tessdata']
+                tessdata_path = '/'.join(path)
+                api = PyTessBaseAPI(path=tessdata_path)
+            except RuntimeError:
+                raise RuntimeError('Please install tesseract first. '
+                                   'Check out the installation guide at'
+                                   'https://github.com/sirfz/tesserocr')
```

Comment:
May be the info shoud be:
```python
raise RuntimeError(' Please download some .tessdata first. '
                               ' 1. Check https://github.com/tesseract-ocr/tessdata '
                               ' 2. Choose the corresponding language file, E.g., to recognize english, 
                                     just download 'eng.traineddata'
                                '3.  Creat a folder named `tessdata` in current path and put the `.tessdata` file in it'
```
Forgive my poor Review code

Reply:
My bad, `tessdata` is a wrapper of `tesseract', so, it should be:
```Python
                raise RuntimeError('Please install tesseract first. '
                                   'Check out the installation guide at'
                                   'https://github.com/UB-Mannheim/tesseract/wiki')
```

Reply:
After download [tesseract](https://github.com/UB-Mannheim/tesseract), `tessdata` would be located near `tesseract.exe` which can detected by shell command 'where tesseract'

Reply:
Sounds good